sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/sas/TableAccountSasResourceType.java

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables.sas;
+
+import com.azure.data.tables.implementation.StorageConstants;
+
+import java.util.Locale;
+
+/**
+ * This is a helper class to construct a string representing the resources accessible by an Account SAS. Setting a value
+ * to true means that any SAS which uses these permissions will grant access to that resource type. Once all the values
+ * are set, this should be serialized with {@code toString()} and set as the resources field on an
+ * {@link TableAccountSasSignatureValues} object. It is possible to construct the resources string without this class,
+ * but the order of the resources is particular and this class guarantees correctness.
+ */
+public final class TableAccountSasResourceType {
```

Comment:
I think this type should be removed in favor of `AccountSasResourceType` in `azure-storage-common`.

Reply:
Please see my comment above.